modules/plugins/languages/yaml.nix

```
@@ -14,16 +15,17 @@
 
   cfg = config.vim.languages.yaml;
 
-  onAttach =
-    if config.vim.languages.helm.lsp.enable
+  on_attach = mkLuaInline (
+    if config.vim.languages.helm.lsp.enable && config.vim.languages.helm.enable
     then ''
-      on_attach = function(client, bufnr)
+      function(client, bufnr)
```

```suggestion
      function(client, bufnr)
        default_on_attach()
```
call default_on_attach otherwise LSP keybinds (and other stuff) will be unavailable